tests/example.spec.ts

```diff
@@ -1,24 +1,24 @@
-import { test, expect } from "@playwright/test";
+import { test, expect } from '@playwright/test'
 
-test("has hello", async ({ page }) => {
-  await page.goto("http://localhost:3000");
+test('has hello', async ({ page }) => {
+  await page.goto('http://localhost:3000')
 
-  await expect(page.getByText("Hello", { exact: true })).toBeVisible();
-});
+  await expect(page.getByText('Hello', { exact: true })).toBeVisible()
+})
+
+test('has world', async ({ page }) => {
+  await page.goto('http://localhost:3000')
+
+  await page.waitForFunction(() => window.msw)
```

Comment:
2: Await `window.msw` to be set by your code. `enableMocking` is async, so let's not create race conditions between it and your test accessing `window.msw`.